python/python/bystro/proteomics/tests/test_annotation_interface.py

```diff
@@ -55,3 +60,38 @@ def tests__process_response():
     expected_dosage_values = {1.0, 2.0, MISSING_GENO_VALUE}
     actual_dosage_values = set(ans.dosage.fillna(MISSING_GENO_VALUE).unique())
     assert expected_dosage_values == actual_dosage_values
+
+
+def test_join_annotation_results_to_proteomics_dataset():
+    # Step 1: Get an annotation query result
+    user_query_string = "exonic (gnomad.genomes.af:<0.1 || gnomad.exomes.af:<0.1)"
+    index_name = None
+    mock_client = MockOpenSearch()
+    query_result_df = get_samples_and_genes_from_query(user_query_string, index_name, mock_client)
+
+    # Step 2: Construct a proteomics dataset with some shared sampled_ids and gene_names
```

Comment:
We may want to make a note somewhere that in future we'll have to take in a third kind of 'tracking ID' that will connect proteomic and genomic sample_ids

Reply:
Good point, do we have a sense now of how we want that tracking ID to work?  Does the relationship look like:

sample -> tracking_id -> (genomic_id, proteomic_id)?

i.e. a sample is associated with a unique tracking_id, and each tracking_id has a unique genomic_id and a unique_proteomic_id that are not necessarily equal to each other?

Reply:
Yes, I believe that's correct.

Reply:
ok, added in the plumbing to map genomic_ids and proteomic_ids to tracking_ids in the future